[KOGITO-9324] -  Testing CLI + Operator using serverless-workflow-foreach-quarkus (#1687)
kiegroup/kogito-examples: serverless-workflow-examples/serverless-workflow-foreach-quarkus/README.md

 }
```

### Building and Deploying Workflow using CLI + Kogito Serverless Workflow Operator

> For this prepare your environment by following the instructions from [here]().

Reviewer: missing link ?

Author: @radtriste The mentioned doc is not yet created. We need to update it later when its created.

> Refer to [Serverless Workflow Guide](), to know how to build and deploy workflows using CLI + Kogito Serverless Workflow Operator.

Reviewer: link ?

Author: @radtriste The mentioned doc is not yet created. We need to update it later when its created.

 }
```

### Building and Deploying Workflow using Kogito Serverless Workflow Operator

> For this install Kogito Serverless Workflow Operator in your Kubernetes cluster from [here](https://kiegroup.github.io/kogito-docs/serverlessworkflow/latest/cloud/operator/install-serverless-operator.html).

Reviewer: ```suggestion
You need to install the Kogito Serverless Workflow Operator in your Kubernetes cluster from [here](https://kiegroup.github.io/kogito-docs/serverlessworkflow/latest/cloud/operator/install-serverless-operator.html).
```

In native Kubernetes approach you can model workflows using [YAML](operator/foreach_sw.yaml) definitions and directly deploy them using the Kogito Serverless Workflow Operator.

> Refer to [Serverless Workflow Guide](https://kiegroup.github.io/kogito-docs/serverlessworkflow/latest/cloud/index.html), to know how to build and deploy workflows using Kogito Serverless Workflow Operator.

Reviewer: ```suggestion
Refer to the [Serverless Workflow Guide](https://kiegroup.github.io/kogito-docs/serverlessworkflow/latest/cloud/index.html) to know how to build and deploy workflows using Kogito Serverless Workflow Operator.
```
